DuckDuckGo/PrivateSearchView.swift

```diff
@@ -58,26 +58,32 @@ struct PrivateSearchViewSettings: View {
             // Autocomplete Suggestions
             SettingsCellView(label: UserText.settingsAutocomplete,
                              accesory: .toggle(isOn: viewModel.autocompletePrivateSearchBinding))
+
+            if viewModel.shouldShowRecentlyVisitedSites {
+                SettingsCellView(label: UserText.settingsAutocompleteRecentlyVisited,
+                                 accesory: .toggle(isOn: viewModel.autocompleteRecentlyVisitedSitesBinding))
+            }
+
         }
 
-        Section(footer: Text(UserText.voiceSearchFooter)) {
-            // Private Voice Search
-            if viewModel.state.speechRecognitionAvailable {
+        if viewModel.state.speechRecognitionAvailable {
```

Comment:
Fixes bug here https://app.asana.com/0/392891325557410/1207368549625525/f 

ie section footer doesn't get hidden if microphone is not available